src/mds/Server.h

```diff
@@ -144,6 +144,7 @@ class Server {
   void submit_mdlog_entry(LogEvent *le, MDSLogContextBase *fin,
                           MDRequestRef& mdr, const char *evt);
   void dispatch_client_request(MDRequestRef& mdr);
+  void perf_gather_op_latency(__u32 op, utime_t lat);
```

Comment:
The prototype needs its arguments updated too.

Reply:
sorry